public/javascripts/rating.js

```diff
@@ -0,0 +1,86 @@
+const checkRating = async () => {
+    const movieId = document.getElementsByClassName('stars')[0].id;
+    const res = await fetch(`http://localhost:8080/reviews/rating/${movieId}`);
+    
+    const {rating} = await res.json();
+    if(rating){
+        for(let i = 1; i <= rating.rating; i++){
+            const el = document.getElementById(`star-rating_${i}`)
+            el.classList.remove('far')
+            el.classList.add('fas')
```

Comment:
Can you please tell me where the `far` and `fas` classes are defined and what thy are doing?

Reply:
Far is grabbing the 'regular' favicon from font awesome (not filled in) Fas is grabbing the solid favicon for the filled in. then in css i am coloring it with .fa-star{} rather than the default black

Reply:
ohh I see, yeah that totally makes sense. Thanks for clarifying.